db/migrate/20160712045614_create_tasks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTasks < ActiveRecord::Migration
+  def change
+    create_table :tasks do |t|
+      t.string :name
+      t.index :name, unique: true
+      t.text :description
+      t.integer :state, default: -1
+      t.references :project
```

Comment:
It's cool to use `t.references` but it would be good to give it some options, especially the option that tells it that the reference is a `uuid` but also adding an index is probably good since your service does a query based on it and for thoroughness tossing in a a foreign key option might be good.